modules/php/Managers/Cards.php

```diff
@@ -165,4 +165,17 @@ public static function stole($mixed, $player)
     $cId = is_int($mixed) ? $mixed : $mixed->getId();
     Cards::move($cId, ['hand', $player->getId()]);
   }
+
+  public static function createSelection($nbr, $player = PUBLIC_SELECTION) {
+    self::moveAllInLocation('selection', 'discard');
+    return self::pickForLocation($nbr, 'deck', 'selection');
+  }
+
+  public static function getSelection() {
```

Comment:
I think this function should return the collection rather than the array so that we can use it in other locations of the code.
Also the line that should never happen does not seems consistent with the return type of the general case (array of cards VS array with a 'player' field)

Reply:
Removed whole "should never happen" block as it might fix some bug which doesn't exist anymore